### DISCUSSION TITLE
自有域名搭建群晖5001隧道开启https失败，需要指导

### DISCUSSION THREAD

**Discussion by chouqiao** [natfrp/wiki#398] [使用问题]
计划用自己的域名aaa.bbb.com访问内网群晖，按照教程搭建，可以访问，但无法通过https访问，请求指导

域名已备案，托管在腾讯dnspod，计划用二级域名aaa.bbb.com访问，已经在管理面板添加cname记录指向隧道网址。

黑群使用7.0.1版本，使用docker最新镜像安装。

隧道管理面板添加隧道，添加隧道-https建站隧道进行添加。关闭自动https。

群晖docker添加容器，储存空间选项映射二级域名的key和crt文件，启动容器。

用edge浏览器访问aaa.bbb.com，显示503 Service Unavailable
访问https://aaa.bbb.com edge浏览器显示"你的连接不是专用连接",点击高级后选择继续访问，可以用不安全的http方式访问群晖。

我哪里设置的不对？请大佬们指点

我也停止了容器，重新把两个证书文件复制到映射的文件夹里。 启动后问题依旧/

**Comment by chouqiao** [comment]
严格按照教程操作
1、https://doc.natfrp.com/app/synology.html  用这个教程安装，用不隧道提供的域名+端口访问（可访问，但不能进行https访问）
2、https://doc.natfrp.com/frpc/ssl.html#autohttps-sslfile 使用这个教程更改为自有域名进行https访问，不会用群晖docker的命令行，使用面板的储存空间挂载两个证书文件。
浏览器访问aaa.bbb.com  显示503 Service Unavailable #25
加https 访问 显示此站点的连接不安全

**Reply by chouqiao** [reply]
火狐浏览器https访问显示建立安全连接失败

连接到aaa.bbb.com 时发生错误。SSL 接收到一个超出最大准许长度的记录。

**Comment by fakeboboliu** [comment]


**Reply by chouqiao** [reply]
感谢回复，我发邮件给您

**Reply by chouqiao** [reply]
通过指导，已经设置成功，谢谢管理员

**Comment by yjm419** [comment]
你好，我也是同样问题，请问怎么配置好的
